ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/tunnel-interfaces.j2

```diff
@@ -0,0 +1,79 @@
+{% if tunnel_interfaces is defined %}
+
+## Tunnel Interfaces
+
+### Tunnel Interfaces Summary
+
+| Interface | Description | VRF | MTU | Shutdown |
+| --------- | ----------- | --- | --- | -------- |
+{%     for tunnel_interface in tunnel_interfaces | arista.avd.natural_sort('name') %}
+{%         set description = tunnel_interface.description | arista.avd.default('-') %}
+{%         set vrf = tunnel_interface.vrf | arista.avd.default('default') %}
+{%         set mtu = tunnel_interface.mtu | arista.avd.default('-') %}
+{%         set shutdown = tunnel_interface.shutdown | arista.avd.default('-') %}
+| {{ tunnel_interface.name }} | {{ description }} | {{ vrf }} | {{ mtu }} | {{ shutdown }} |
+{%     endfor %}
+
+{# IPv4 #}
+{%     set tunnel_interfaces_ipv4 = [] %}
+{%     for tunnel_interface in tunnel_interfaces | arista.avd.default([]) %}
+{%         if tunnel_interface.ip_address is arista.avd.defined %}
+{%             do tunnel_interfaces_ipv4.append(tunnel_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if tunnel_interfaces_ipv4 | length > 0 %}
+#### IPv4
```

Comment:
Please add space before optional sections, and remove extra spaces after previous sections. This way we have consistent spacing in all combinations.